src/components/atoms/_timeline.scss

```
@@ -0,0 +1,85 @@
+.tk-timeline {
+  display: flex;
+  flex-direction: column;
+  overflow-y: scroll;
+  overflow-x: hidden;
+
+  &__item {
+    display: flex;
+    flex-direction: row;
+    margin-left: toRem(10);
+    padding-bottom: toRem(12);
+
+    &-isExpandable {
```

```suggestion
    &--isExpandable {
```

for consistency with other parts of the library.

---

Not just `--isExpandable` but maybe all usage of single "-" or "_".